src/condor_tools/htcondor_cli/convert_ad.py

```diff
@@ -0,0 +1,127 @@
+from htcondor._utils.ansi import Color, colorize
+from time import time
+
+DAEMON_HP_DEFAULT = -1
+DAEMON_MP_MAX = 100
```

Comment:
I'm not 100% sold on "HP" as a metric, rather than a more heuristic approach, but if you're going to, at least be consistent; none of our daemons our magic, they don't have any MP.  Just HP.

Reply:
You sure... I think the `master` daemon is a necromancer given its affinity to raise killed `daemons`